README.md

```diff
@@ -1,31 +1,34 @@
 <a href="https://lacework.com"><img src="https://techally-content.s3-us-west-1.amazonaws.com/public-content/lacework_logo_full.png" width="600"></a>
 
-# Lacework Provisioning with Terraform
+# DEPRECATED - Lacework Provisioning with Terraform
 
 [![GitHub release](https://img.shields.io/github/release/lacework/terraform-provisioning.svg)](https://github.com/lacework/terraform-provisioning/releases/)
 [![CircleCI status](https://circleci.com/gh/lacework/terraform-provisioning.svg?style=shield)](https://circleci.com/gh/lacework/terraform-provisioning)
 
-This repository contains [Terraform](https://terraform.io) code for provisioning resources
-required to integrate public cloud environments (AWS, GCP, Azure) into Lacework's automated
-security platform.
+## **WARNING - THIS PROJECT IS BEING DEPRECATED**
 
-For more information visit [support.lacework.com](https://support.lacework.com/).
+This project is being deprecated. All modules developed in this repository have been migrated to separate github repositories, and are being published directly to the [Terraform Registry](https://registry.terraform.io/search/modules?q=lacework). This migration provides support for Terraform `0.13` and allows Lacework to release new features and fixes much faster!
 
-## Requirements
-- [Terraform](https://terraform.io) 0.12.x
+### New Project Locations
+* **AWS**
```

Comment:
```suggestion
* **AWS Modules**
```

Reply:
@scottford-lw what do you think about adding "Modules" to these sections? If not, just ignore them.